test/test_helper.rb

```diff
@@ -4,15 +4,15 @@
 
 class ActiveSupport::TestCase
   # Setup all fixtures in test/fixtures/*.yml for all tests in alphabetical order.
-  fixtures :all     
+  fixtures :all
 
   Time.zone = 'Alaska'
   Time.now
   Date.today
-begin
-  
-rescue Exception
-  
-end
+  begin
+
+  rescue Exception
```

Comment:
__[rubocop]__
* Do not suppress exceptions. [:ref](https://github.com/bbatsov/ruby-style-guide#dont-hide-exceptions)
* Avoid rescuing the `Exception` class. Perhaps you meant to rescue `StandardError`? [:ref](https://github.com/bbatsov/ruby-style-guide#no-blind-rescues)